src/models/departmentDetails.model.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (sequelize, Sequelize) => sequelize.define('departmentDetails', {
+    
+    id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+    min_income: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        },
+    max_income: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        },
+    introduced_date:{
+        type: Sequelize.DATEONLY,
```

Comment:
I am ot sure about DATEONLY, please cross check with boilerplate

Reply:
I want only date to be returned for SQL so i have used DATEONLY which will return YYYY-MM-DD in this formate which is one type of ISO formate do not need time if we wanted along with the time then i would have used DATE as type which will return YYYY-MM-DDTHH:MM:SSZ